make/ci.mk

```diff
@@ -1,6 +1,10 @@
 .PHONY: ci-presubmit
-ci-presubmit: verify-imports
+ci-presubmit: verify-imports verify-chart
 
 .PHONY: verify-imports
 verify-imports: bin/tools/goimports
 	./hack/verify-goimports.sh $<
+
+.PHONY: verify-chart
+verify-chart: bin/cert-manager-$(RELEASE_VERSION).tgz
+	./hack/verify-chart-version.sh $<
```

Comment:
Probably unrelated to your change: I noticed many "in the future" lines:

```
$ make -f make/Makefile verify-chart
...
Linting chart 'bin/cert-manager-v1.7.0-alpha.0-67-g72027ce62.tgz' using internal dir 'deploy/charts/cert-manager'
cert-manager/Chart.yaml
tar: cert-manager/Chart.yaml: time stamp 2022-01-10 20:10:55 is 0.29450188 s in the future
cert-manager/values.yaml
tar: cert-manager/values.yaml: time stamp 2022-01-10 20:10:55 is 0.294457475 s in the future
cert-manager/templates/NOTES.txt
tar: cert-manager/templates/NOTES.txt: time stamp 2022-01-10 20:10:55 is 0.294428667 s in the future
cert-manager/templates/_helpers.tpl
tar: cert-manager/templates/_helpers.tpl: time stamp 2022-01-10 20:10:55 is 0.294409669 s in the future
cert-manager/templates/cainjector-deployment.yaml
```

I guess that's something that isn't to be worried about.

Reply:
The verification always runs in a docker container and I believe messages like this are related to docker. Longer term I'd like to remove that dependency on docker for doing this verification.